pom.xml

```diff
@@ -46,70 +46,69 @@
             <artifactId>asciidoctorj</artifactId>
             <version>1.5.8.1</version>
```

Comment:
While we're at it and doing a full 2.x rev, we should really sync with the rest of the asciidoctor projects and use the same versions. That would mean upgrading to the latest asciidoctorj. I know that's a lot for a single pull request though.

Reply:
This is a good version policy to have. If it needs to be done in this PR, then I'd like to get the Java 11 port working first, though, with minimal behavioural changes.

Reply:
+1